examples/sensor/README.md

```diff
@@ -0,0 +1,20 @@
+(sensor)=
+
+# Sensor
+
+```{eval-rst}
+.. tags:: Data, Advanced
+```
+
+## Run the example on the Flyte cluster
+
+To run the provided example on the Flyte cluster, use the following command:
+
+```
+pyflyte run --remote \
+  https://raw.githubusercontent.com/flyteorg/flytesnacks/master/examples/sensor/sensor/sensor.py wf
```

Comment:
why do we need two sensor folder. `/sensor/sensor/...`? can we update the file structure here, update it to
```
- sensor
| _ file_sensor_example.py
| _ http_sensor_example.py
| _ etc...
```

Reply:
I think all the other examples are like this, and have a README and Dockerfile in the top-level directory, and an __init__.py and example file in the lower-level one. This one appears to be missing a Dockerfile.